sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosContainerProperties.java

```diff
@@ -178,6 +179,27 @@ public CosmosContainerProperties setChangeFeedPolicy(ChangeFeedPolicy value) {
         return this;
     }
 
+
+    /**
+     * Gets the computedProperties for this container in the Azure Cosmos DB service.
+     *
+     * @return the computedProperties.
+     */
+    public Collection<ComputedProperty> getComputedProperties() {
+        return this.documentCollection.getComputedProperties();
+    }
+
+    /**
+     * Sets the computedProperties for this container in the Azure Cosmos DB service.
+     *
+     * @param computedProperties the computedProperties.
+     * @return the CosmosContainerProperties.
+     */
+    public CosmosContainerProperties setComputedProperties(Collection<ComputedProperty> computedProperties) {
+        this.documentCollection.setComputedProperties(computedProperties);
```

Comment:
validations missing? Can computedProperties be ever null? What if the list contains null values etc.

Reply:
The check is inside the documentCollection class. I've added check for the presence of null values also